flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptive/AdaptiveScheduler.java

```diff
@@ -145,14 +151,14 @@
  */
 public class AdaptiveScheduler
         implements SchedulerNG,
-                Created.Context,
-                WaitingForResources.Context,
-                CreatingExecutionGraph.Context,
-                Executing.Context,
-                Restarting.Context,
-                Failing.Context,
-                Finished.Context,
-                StopWithSavepoint.Context {
+        Created.Context,
+        WaitingForResources.Context,
+        CreatingExecutionGraph.Context,
+        Executing.Context,
+        Restarting.Context,
+        Failing.Context,
+        Finished.Context,
+        StopWithSavepoint.Context {
```

Comment:
Hi @bytesandwich , could you update your IDE's project settings following the [code formatting setup recommended in the Flink docs|https://ci.apache.org/projects/flink/flink-docs-master/docs/flinkdev/ide_setup/#code-formatting]? That would avoid changes like that to happen. I'm gonna have a look at the PR after you updated it.

Reply:
hi @XComp thanks for pointing that out. my google-java-format plugin was enabled in the plugin install page but was disabled in its own settings, so going through the steps fixed it and removed a lot of noise.

Here are some things you might also consider in your review:

- Maybe Executing and StopWithSavepoint shouldn't always call archiveExecutionFailure as I have in the PR?

- Should we rename the class `FailureHandlingResultSnapshot` to `FailureSnapshot` or something like that?

Will this work? @flinkbot run travis

Reply:
ping @XComp

Reply:
Sorry for the late response. I am still busy with other stuff. I hope to get back to you today or on Monday.